cmd/gen_test.go

```diff
@@ -71,10 +74,16 @@ var _ = Describe("Commands", func() {
 			platform.PlatformConfig = config
 		})
 		It("Generate Mtar", func() {
-			os.MkdirAll(getTestPath("result", ".mtahtml5_mta_build_tmp", "testapp"), os.ModePerm)
-			os.MkdirAll(getTestPath("result", ".mtahtml5_mta_build_tmp", "ui5app2"), os.ModePerm)
+			dir.CreateDirIfNotExist(getTestPath("result", ".mtahtml5_mta_build_tmp", "testapp"))
```

Comment:
no, mtar is created from the ready tmp folder, created on the previous steps (build module, etc..)

Reply:
Yes - but the "testapp" folder shouldn't be in the mtar since it's just the module path. The build artifact for the module is under the module's name, not path, right?

Reply:
yes, removed